tests/suites/os/tests/modem/index.js

```diff
@@ -18,7 +18,7 @@ const fsPromises = require('fs').promises;
 const CONFIG_FILE = `${__dirname}/modems.json`;
 
 const SCAN_ATTEMPTS = 5;
-const PING_ATTEMPTS = 3;
+const CURL_ATTEMPTS = 3;
```

Comment:
Since we are using cURL to effectively perform a non-ICMP ping, I kind of prefer we keep the `PING_ATTEMPTS` and related terminology.

Reply:
Good idea, I will add a comment for this as well.